app/__tests__/unit/root.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { expect, describe, it } from "vitest";
 import { loader, action } from "~/root";
-import { prefs } from "~/prefs-cookie";
+import { userPrefs } from "~/cookies.server";
```

Comment:
More descriptive naming of the purpose of this cookie (reight now it persists their selected theme between page refresh / visits), `.server` tells remix framework this code will only run on the server.